regression/cbmc/Quantifiers-if/main.c

```diff
@@ -9,16 +9,16 @@ int main()
   if(__CPROVER_forall { int i; (i>=0 && i<2) ==> a[i]>=1 && a[i]<=10 })
     __CPROVER_assert(0, "failure 1");
 
-  if(__CPROVER_exists { int i; (i>=0 && i<2) ==> a[i]>=1 && a[i]<=10 })
+  if(__CPROVER_exists { int i; (i>=0 && i<2) && a[i]>=1 && a[i]<=10 })
```

Comment:
These seem to change the semantics of the test -- it's not really Quantifiers-if is it?

Reply:
It's "quantifiers used within `if`" so I think that's ok? And re changing their semantics: I think `==>` may have been misused, the `&&` should actually capture the intention (bounded quantification) here.

Reply:
OK.